.github/ISSUE_TEMPLATE/design-intent-collaboration.md

```diff
@@ -31,8 +31,8 @@ assignees:
 - [ ] Close this ticket when review is complete and feedback has been addressed. 
 
 
-### Has your product been reviewed by the OCTO-DE Design Lead?
-- [ ] Yes
+### Has your product been reviewed by the OCTO-DE?
```

Comment:
I'm not sure we want to leave this in bc many VFS teams (like VSA, VAOS, etc.) have OCTO-DE as their PO. For example, Samara, Matt Self, Dave Conlon, etc. are all OCTO-DE. If I were VFS, and I saw this question, I'd answer yes because Samara (my PO in this example) has viewed it. What we really want to know is if someone in a practice area lead has reviewed the product.... someone subjective who doesn't have ownership in the product. Currently these people are Martha, Beth, Danielle, Kevin maybe. 

I see what you're getting at, but I think for now the answer is to just remove it. If we really think it's valuable to know, maybe we just add it to the script and ask in person at the meeting.